runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -228,13 +228,13 @@ public void render(Graphics2D graphics)
 							bottomLeftPoint.x += dimension.width + (dimension.width == 0 ? 0 : PADDING);
 							break;
 						case BOTTOM_RIGHT:
-							bottomRightPoint.x -= dimension.width - (dimension.width == 0 ? 0 : PADDING);
+							bottomRightPoint.x -= dimension.width - (dimension.width == 0 ? 0 : -PADDING);
```

Comment:
Double negation, the logic can be changed from d.widh - (-PADDING) to d.width + PADDING